client/me/purchases/billing-history/receipt.tsx

```diff
@@ -138,27 +138,32 @@ export function ReceiptBody( {
 	const moment = useLocalizedMoment();
 	const title = translate( 'Visit %(url)s', { args: { url: transaction.url }, textOnly: true } );
 	const serviceLink = <a href={ transaction.url } title={ title } />;
+	const headerByLine = 'memberships' === transaction.service_slug ? 'Payment processed by' : 'by';
```

Comment:
These strings should probably also be translated. I'm not sure how to translate the standalone `by` 🤔. Also, some languages might put the thing you're processing by _before_ the "by". It's probably a better idea to translate the full string, even though it means you'll need to have two fully separate translations.

Reply:
>It's probably a better idea to translate the full string

Yeah, that way would also preserve the existing `'{{link}}%(service)s{{/link}} {{small}}by %(organization)s{{/small}}'` translation as-is (so it's really only adding one new translation anyway).

Reply:
Thanks for the review. I have fixed the translation strings. Putting string freeze on the PR while I wait for a final review.